moveit_ros/moveit_servo/src/teleop_demo/joystick_servo_example.cpp

```diff
@@ -91,8 +95,8 @@ enum Button
 
 // Some axes have offsets (e.g. the default trigger position is 1.0 not 0)
 // This will map the default values for the axes
-std::map<Axis, double> AXIS_DEFAULTS = { { LEFT_TRIGGER, 1.0 }, { RIGHT_TRIGGER, 1.0 } };
```

Comment:
What is the reason for this change?

Reply:
Naming convention, I assume. This is a local variable, not a constant.

Reply:
i'll push to fix this. Also, the next variable `button_defaults` is unused.

Reply:
It was clang-tidy related